### PR TITLE
OP executable proposals: required app upgrades

### PR DESCRIPTION
**Pull request by mcgingras**
## Prepares app for OP executable proposals test
1. replaces PTC testnet address
2. pays down tech debt to allow tenant config to define which voting modules it supports, and the addresses of each of those modules. Previously, the addresses of the modules were hardcoded, and we were supporting only OP mainnet and cyber testnet (???). Now these are configurable at the tenant level, and will throw errors if not configured.

Since 2) changes PLM code, it would be great if we could get a few QA passes done. All we really need to test is that we can create proposals for each supposed voting module on each tenant. After the proposal is made, it's a different code path that these changes don't touch.

### CODE REVIEW

File: src/app/proposals/draft/[id]/page.tsx [hunk 1]
```diff
@@ -74,14 +74,14 @@ export default async function DraftProposalPage({
           </section>
           <section className="col-span-1">
             <div className="bg-wash border border-line rounded-2xl p-4">
-              <p className="mt-2">
+              <div className="mt-2">
```
Comment (mcgingras) [inline src/app/proposals/draft/[id]/page.tsx:77]: React was complaining about nesting a dev in a p tag... simple to change, doesn't matter much

File: src/lib/tenant/configs/ui/ens.ts [hunk 2]
```diff
@@ -132,7 +132,18 @@ export const ensTenantUIConfig = new TenantUI({
...
+            type: ProposalType.BASIC,
+            prodAddress: null,
+            testnetAddress: null,
+          },
+          {
+            type: ProposalType.SOCIAL,
+            prodAddress: null,
+            testnetAddress: null,
+          },
+        ],
```
Comment (mcgingras) [inline src/lib/tenant/configs/ui/ens.ts:146]: basic and social type do not need to define any address -- there is no module to support for basic or social

File: src/components/Proposals/ProposalPage/OPProposalPage/OPProposalOptimisticPage.jsx [hunk 3]
```diff
@@ -78,16 +77,13 @@ export default async function OPProposalPage({ proposal }) {
...
-    // 2 Colum Layout: Description on left w/ transactions and Votes / voting on the right
-    <HStack
-      gap={16}
-      justifyContent="justify-between"
-      alignItems="items-start"
-      className="max-w-[76rem] flex-col sm:flex-row items-stretch sm:items-start justify-end sm:justify-between"
-    >
-      <ProposalDescription proposalVotes={proposalVotes} proposal={proposal} />
-      <div>
-        <StandardProposalDelete proposal={proposal} />
```
Comment (andreitr) [inline src/components/Proposals/ProposalPage/OPProposalPage/OPProposalOptimisticPage.jsx:90]: Removed StandardProposalDelete

**Review by andreitr — APPROVED** [review]
Looks LGTM

**Pull request merged**